icons.md

```diff
@@ -21,11 +21,35 @@ layers:
     my-layer:
         data: { source: my-source }
         draw:
-            icons:
+            mapzen_icon_library:
```

Comment:
We should have two examples, one that is:

```
         draw:
            mapzen_icon_library:
               sprite: zoo
```

```
Second that says if your data has a kind property with one of the values in the table below, it'll just show up:

```
         draw:
            mapzen_icon_library: {}
```